tools/InputLatency.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+
+import pandas as pd
+import numpy as np
```

Comment:
It's ok. Just put there as template combination. Pandas, numpy, and matplotlib always works together